lib/operation-helper.specs.js

```diff
@@ -148,6 +148,101 @@ describe('OperationHelper', function () {
         })
     })
 
+    describe('#getQueueSize', () => {
+      let opHelper, actual
+
+      context('no throttling', () => {
+        before(() => {
+          opHelper = new OperationHelper({
+              awsId: 'testAwsId',
+              awsSecret: 'testAwsSecret',
+              assocId: 'testAssocId',
+              maxRequestsPerSecond: 0
+          })
+
+          opHelper.execute('ItemSearch', {
+              'SearchIndex': 'Books',
+              'Keywords': 'harry potter',
+              'ResponseGroup': 'ItemAttributes,Offers'
+          })
+
+          actual = opHelper.getQueueSize()
+        })
+
+        it('should return -1', () => {
+            expect(actual).to.equal(-1)
+        })
+      })
+
+      context('throttling, but with an empty queue', () => {
+        before(() => {
+          opHelper = new OperationHelper({
+              awsId: 'testAwsId',
+              awsSecret: 'testAwsSecret',
+              assocId: 'testAssocId',
+              maxRequestsPerSecond: 1
+          })
+
+          actual = opHelper.getQueueSize()
+        })
+
+        it('should return 0', () => {
+            expect(actual).to.equal(0)
+        })
+      })
+
+      context('throttling with completed queue', () => {
+        before(() => {
+          opHelper = new OperationHelper({
+              awsId: 'testAwsId',
+              awsSecret: 'testAwsSecret',
+              assocId: 'testAssocId',
+              maxRequestsPerSecond: 1
+          })
+
+          opHelper.execute('ItemSearch', {
```

Comment:
Cool, I've started on writing tests for Throttler in lib/throttler.specs.js 

It also includes tests for .getQueueSize()